crates/neon/src/lib.rs

```diff
@@ -143,18 +143,21 @@ pub struct Exports(());
 impl Exports {
     /// Export all values exported with [`neon::export`](export)
     ///
-    /// ```ignore
+    /// ```
+    /// # fn main() {
     /// # use neon::prelude::*;
     /// #[neon::main]
     /// fn main(mut cx: ModuleContext) -> NeonResult<()> {
     ///     neon::registered().export(&mut cx)?;
     ///     Ok(())
     /// }
+    /// # }
     /// ```
     ///
     /// For more control, iterate over exports.
     ///
-    /// ```ignore
```

Comment:
On my laptop, this test takes 9s with a hot cache and 45s with a completely cold cache.

While that's slower than I'd like, I feel like that's fast enough to include by default. I've caught regressions in feature flag combos a few times. What do you think?